backend/portal/models.py

```diff
@@ -48,17 +50,54 @@ class Content(models.Model):
     admin_comment = models.CharField(max_length=255, null=True, blank=True)
     target_populations = models.ManyToManyField(TargetPopulation, blank=True)
     priority = models.IntegerField(default=0)
+    creator = models.ForeignKey(User, on_delete=models.SET_NULL, null=True, blank=True)
 
     class Meta:
         abstract = True
 
+    def _get_email_subject(self):
+        return f"[Portal] {self.__class__._meta.model_name.capitalize()} #{self.id}"
+
+    def _on_create(self):
+        send_automated_email.delay_on_commit(
+            self._get_email_subject(),
+            get_backend_manager_emails(),
+            (
+                f"A new {self.__class__._meta.model_name} for {self.club_code}"
+                f"has been created by {self.creator}."
+            ),
+        )
+
+    def _on_status_change(self):
+        if email := getattr(self.creator, "email", None):
```

Comment:
don't need to pass in the last param for `getattr` since the default return value if not found is `None`.

also... should do `if _ is not None` ... more Pythonic :)

Reply:
no u need the default else it errors. 

but we have this pattern everywhere in our code...

Reply:
o u right

argh fine keep it